components/demo/demo-page.js

```diff
@@ -80,7 +80,8 @@ class DemoPage extends LitElement {
 		const classes = {
 			'no-scroll': this._noScroll
 		};
-		const selectedLanguageCode = getDocumentLocaleSettings().language;
+		let selectedLanguageCode = getDocumentLocaleSettings().language;
+		if (selectedLanguageCode === 'en') selectedLanguageCode = 'en-us';
```

Comment:
The way the language selector in the demo page works, it goes through all the options and finds the first partial match. But because the page's language is `en` it was matching `en-ca` even though the components will interpret that as `en-us`.